bin/setup

```diff
@@ -1,14 +1,18 @@
-#!/bin/sh
+#!/usr/bin/env sh
```

Comment:
Is this for Solaris 8?

Reply:
Yes? I don't actually know why this got changed, it's possible I copied someone else's setup script and this happened.